packages/js-recommendations/src/types/RenderProps.ts

```diff
@@ -0,0 +1,4 @@
+export type RenderProps = {
```

Comment:
The name `RenderProps` if a bit confusing. How about `DomProps`, `JsProps` or `EnvironmentProps`?

Reply:
`EnvironmentProps` seems appropriate to me!

Reply:
Done in https://github.com/algolia/react-recommendations/pull/6/commits/1690ca429f4dd755ae80fa7b2eca87aa6bc311c2